### ISSUE TITLE
Can we have  millisecond-level timestamps in argoexec?

### ISSUE THREAD

**Issue by mikhailian** [argoproj/argo#2795]
# Summary

Currently, timestamps are `time="2020-04-23T08:28:23Z"` , it would help to have milliseconds included, e.g. `time="2020-04-23T08:28:23.000Z"` 

# Motivation

argoexec spawns short-lived processes where milliseconds matter.

# Proposal

```
diff --git a/cmd/argoexec/main.go b/cmd/argoexec/main.go
index 99c6b4d3..5de968ab 100644
--- a/cmd/argoexec/main.go
+++ b/cmd/argoexec/main.go
@@ -14,6 +14,11 @@ import (
 )

 func main() {
+  logger := &logrus.Logger{
+      Formatter: &logrus.TextFormatter{
+      TimestampFormat : "2006-01-02 15:04:05.000Z",
+    },
+  }
        if err := commands.NewRootCommand().Execute(); err != nil {
                fmt.Println(err)
                os.Exit(1)

```

**Comment by alexec** [comment]
Would you like to summit a PR?

**Comment by astein-te** [comment]
@alexec I would like to give this a try

**Comment by aistein** [comment]
@alexec I would like to give this a try

**Comment by simster7** [comment]
@aistein Assigned

**Comment by simster7** [comment]
Closed in #2950
